src/RoleObjectFactory.js

```diff
@@ -1,13 +1,14 @@
 import _ from 'lodash';
 import { ROLES } from './Roles';
-import CompositeData from './RoleObjects/CompositeData';
```

Comment:
Just moving this down to line 11 to fit into the order that all the other RoleObjects import use